### DISCUSSION TITLE
[Bug] 谷歌浏览器在使用了:chrome://settings/clearBrowserData 清理浏览数据，会将我所有的回话列表以及相应的设置全部清除

### DISCUSSION THREAD

**Discussion by amDosion** [lobehub/lobehub#2228] [General Question] [answered]
### 💻 系统环境

Windows

### 📦 部署环境

Docker

### 🌐 浏览器

Chrome

### 🐛 问题描述

由于工作的需要，我们需要定期清理浏览器中的项目，
![1](https://github.com/lobehub/lobe-chat/assets/106908523/6538c249-048f-4c8a-b90c-dc1f3264e56c)
![2](https://github.com/lobehub/lobe-chat/assets/106908523/33386a83-7173-451a-9904-9dd1b09deda8)
当我在谷歌浏览器在使用了:chrome://settings/clearBrowserData  清理浏览数据之后，会将我所有的回话列表以及相应的设置全部清除，具体清理的项目在图片中体现。相当于一个全新的界面了.

### 🚦 期望结果

希望能将数据持久化.

### 📷 复现步骤

chrome://settings/clearBrowserData 清理浏览数据之后(默认的选项不变)，会将我所有的回话列表以及相应的设置全部清除

[a comment by an automated account is not shown]

**Reply by ous50** [reply]
看起来是手动清除的，那要不每次清除前备份一下/用WebRTC先同步着？

**Comment by arvinxx** [comment, marked as answer]
本地版数据是持久化在你的浏览器里的，手动清除数据自然会丢失。如果需要持久化数据，可以考虑下我们后续的 cloud 版

[a comment by an automated account is not shown]

**Comment by amDosion** [comment]
在docker中安装，能否使用设置路径的方式来持久化这些数据，而不是在浏览器中？

[a comment by an automated account is not shown]

**Comment by arvinxx** [comment]
> 在docker中安装，能否使用设置路径的方式来持久化这些数据，而不是在浏览器中？

这个应该不行

[a comment by an automated account is not shown]
